app/src/main/java/space/taran/arkshelf/data/LinkRepoImpl.kt

```diff
@@ -26,4 +29,6 @@ class LinkRepoImpl(
         withContext(Dispatchers.IO) {
```

Comment:
Ideally, the function should be declared as `generateResource(resource: Link, rootPath: Path)` if we want to highlight the concepts from `arklib`.